crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -743,7 +743,7 @@ impl Converter for CumulocityConverter {
     ) -> Result<Option<Message>, ConversionError> {
         match message.ops_dir.parent() {
             Some(parent_dir) => {
-                if parent_dir.eq(&self.cfg_dir.join("operations").join("c8y")) {
+                if parent_dir.eq(&self.cfg_dir.join("operations")) {
```

Comment:
Using the `operations` directory instead of the `operation/c8y` surprises me. Are you sure we should be using this, as won't that potentially listen to other non-c8y changes as well?

For example:
`/etc/tedge/operations/az`

Reply:
The `tedge-mapper-c8y` subscribes for notifications for the path `/etc/tedge/operations/c8y`. So, it will receive the notifications only when there is a change in this directory.

Reply:
So why not check the paths of the event against `/etc/tedge/operations/c8y` ?

- if the parent dir of the update file is `/etc/tedge/operations/c8y`, then the event if for the main device.
- if the grand-parent dir of the update file is `/etc/tedge/operations/c8y`, the the event if for a child device.
- otherwise the event has to be ignored.

Assuming that the mapper only subscribed  to `/etc/tedge/operations/c8y` is really fragile.

Reply:
I felt its redundant, if you think it's very fragile, then I am ok to add specific `event` path checks for `parent` n `child` devices.

Reply:
It's fragile because someone else might later add an apparently innocuous new subscriptions and break things in a fully unexpected manner.

Thank you for the update.